src/main/res/values/strings.xml

```diff
@@ -743,6 +743,7 @@
     <string name="pref_show_emails_all">All</string>
     <string name="pref_experimental_features">Experimental Features</string>
     <string name="pref_on_demand_location_streaming">On-demand Location Streaming</string>
+    <string name="pref_realtime_channels">Realtime Channels</string>
```

Comment:
no need for a new string, there is already `enable_realtime`.

(there is a also a hint, but i would not add that, that results in a text desert, if used everywhere. better show an alert explaining some things after enabling, similar to location-streaming or broadcasts. for the explanations in the alert, it is fine if they're english only)

Reply:
thanks, I updated the string and set the summary as well since that was the easiest route,

 about the dialog I am not so sure, and more if it is English only, but if you have a text to recommend share it or push it directly to the PR, I myself don't understand well yet the realtime channels stuff